config/app/app.yaml

```diff
@@ -15,3 +15,19 @@ parameters:
 
     app.testing.site_domain: '%env(TEST_DOMAIN)%'
     ngsite.default.email_change.token_validity_time_minutes: 15
+
+    netgen_user_impersonation.default.content_type_identifier.user_groups: ['user_group']
+    netgen_user_impersonation.default.content_type_identifier.users: ['user']
+    netgen_user_impersonation.default.users.root_location_ids: [5]
+    netgen_user_impersonation.default.users.name_search_fields: ['first_name', 'last_name']
+    netgen_user_impersonation.default.excluded_user_groups:
+        - 13 # administrator users
```

Comment:
Since these are ID's, they should go to `config/app/server`, since I see this as beeing modified based on project requirements.

Reply:
Done - https://github.com/netgen/media-site/pull/194/commits/d4edcafa8f32db685ec444568b26293cf12ed005